### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (1 - Laudits) 

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#480]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

Laudits
1)  Taking a holistic view of accessibility. 
It is good to see a document that looks at accessibility as being a collaborative effort between content creators, user agents, and tools.  This approach can cause some problems around conformance, but I think there is a way around those as described at the end.  I understand how difficult and complicated task it will be to undertake if you try to do anything besides just describe status quo.  Nonetheless, it is a very valuable contribution and kudos for attempting into.  I also recognize that doing it right will considerably slow the process of developing a 3.0.
2)  Better coverage of cognitive language and learning. 
The document also seeks to find a format that will allow better coverage of cognitive language and learning disabilities.  To the extent that this document can do that, then it (along with the new W3C report specifically on cognitive language and learning disabilities) can provide greatly needed contributions to understanding these disabilities and documenting different strategies for addressing those of us experiencing them.
3)  Outcome orientation 
The document describes itself as being outcome oriented.  This is excellent.  This is what WCAG 2.0 also focused on.  That is, it focused on “what should be achieved” but not “how to achieve it” (except for the techniques doc of course).  This (outcome) approach allows for the greatest interplay between content developers and user agents, dividing the load between them and allowing for advances in user agents to remove requirements on authors - without rewriting guidelines.  (For example, a criterion that says that text must be “able to be enlarged” by some factor talks about an outcome not an imperative.  As a result – when browsers built zooming (to that level) into their browsers, authors no longer had to do anything except not break zoom functionality of the browser.  In the future the zoom functionality of the browsers may be even more powerful, further reducing, and perhaps someday eliminating, the need for authors to do anything in order to have pages that will re-layout and reflow as users zoom them.  
Thus, keeping the provisions outcome-oriented is a way of both providing authors with the greatest flexibility in terms of meeting the provisions, but also making the provisions more future proof, or at least future resilient.  
Making outcome-oriented provisions, however, takes a lot of thinking and work.  It is noted here that even with the small sample of 3 provisions provided, only two are outcome-oriented.  It is very easy to slip into the imperative or command or directive format that dictates “what authors should do” rather than “what should be achieved”

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by jspellman** [comment]
Notes for the response on item 3:  Good point and we have to keep the difference in language clear.  The guidelines are imperative and outcomes need to be "what is achieved".  This needs to be added to the style guide.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
